dev/com.ibm.ws.security.jca/src/com/ibm/ws/security/jca/internal/AuthDataServiceImpl.java

```diff
@@ -86,6 +88,14 @@ protected void unsetSecurityService(ServiceReference<SecurityService> reference)
         securityServiceRef.unsetReference(reference);
     }
 
+    protected void setKrb5Service(ServiceReference<KerberosService> ref) {
+        krb5ServiceRef.setReference(ref);
+    }
+
+    protected void unsetKrb5Service(ServiceReference<KerberosService> ref) {
```

Comment:
When we modify the kerberos element or unsetKrb5Service, don't we need to clean up the subject cache?

Reply:
What subject cache are you referring to?

Reply:
Looks like we do not cache the subject and do the Kerberos login every time, right?
NOTE: Caching of subjects was removed under defect 63520, due to the following two reasons
 * which we could not address in the GA timeframe: